### ISSUE TITLE
Request: Add h-feed support

### ISSUE THREAD

**Issue by dissolve** [samuelclay/NewsBlur#1019]
Lots of sites are marking up their pages with microformats2 h-feed and h-entry microformats with higher fidelity (more entries, more content, sooner, more reliable) than legacy XML formats (RSS/Atom).

http://microformats.org/wiki/h-feed
http://microformats.org/wiki/h-entry

It would be great if NewsBlur supported directly subscribing to a people's home pages, and only fallback to legacy feed files if there's no h-feed or h-entry on the home page.

There are microformats2 parsers:
http://microformats.org/wiki/microformats2#Parsers

Thank you

**Comment by jeremycherfas** [comment]
Any progress on this? Even a definitive "No"? It would make a great feed reader even better.

**Comment by samuelclay** [comment]
I agree this would be nice to support. If it's easy we'll get it done soon as I just hired a developer to work on backend code.

**Comment by samuelclay** [comment]
Ideally you'd take this up with the fine folks working on feedparser: https://github.com/kurtmckee/feedparser

**Comment by jeremycherfas** [comment]
Thanks for the suggestion. I looked there and the developer seems to think that parsing h-feed is "out of scope". However, there are forks of feedparser that will do it, and I am in touch with the developer of one of those.
